### DISCUSSION TITLE
Skip Vulnerabilities where status is "fixed"

### DISCUSSION THREAD

**Discussion by matjohnston-r7** [aquasecurity/trivy#8117] [Q&A]
### Question

I was wondering if its possible to skip vulnerabilities with status: fixed being reported during a Trivy Scan?  

### Target

None

### Scanner

Vulnerability

### Output Format

JSON

### Mode

None

**Comment by knqyf263** [comment]
https://trivy.dev/latest/docs/configuration/filtering/#by-status

**Comment by ckcr4lyf** [comment]
@matjohnston-r7 just curious, why would you want to skip the fixed ones? Typically I'd think people would want to skip the unfixed ones.
